src/sentry/static/sentry/app/views/onboarding/project/index.jsx

```diff
@@ -30,31 +42,62 @@ const Project = React.createClass({
     this.setWarning(this.props.name);
     if (this.props.name) this.props.next();
   },
+  renderTeamPicker() {
+    let {team, teams, setTeam} = this.props;
+    if (teams.length < 2) return null;
+    return (
+      <div className="new-project-team">
+        <h4>{t('Team') + ':'}</h4>
+        <div className="project-team-wrapper">
+          <SelectInput
+            value={team}
+            style={{width: 180, padding: '10px'}}
+            required={true}
+            onChange={e => setTeam(e[0].value)}>
+            {teams.map(({slug, name, id}, i) => (
+              <option key={id} value={slug}>{name}</option>
+            ))}
+          </SelectInput>
+        </div>
+      </div>
+    );
+  },
 
   render() {
     return (
       <div className="onboarding-info">
-        <h2>{t('Choose a language or framework')}</h2>
+        <h4>{t('Choose a language or framework' + ':')}</h4>
```

Comment:
`:` is still inside of `t()`

Reply:
you must have been eating your fish eyes